src/Cards/Cards.js

```diff
@@ -30,14 +32,36 @@ export default class Cards extends Component {
     });
   }
 
+  sortMovies = (value) => {
+    if (!value || value === '--none--') {
+      return false
+    } else if (value === 'descending'){
+      const sortedMovies = this.state.movies.sort((a, b) => a.average_rating - b.average_rating)
+      this.setState({ movies: sortedMovies })
+    } else if (value === 'ascending'){
+      const sortedMovies = this.state.movies.sort((a, b) => b.average_rating - a.average_rating)
+      this.setState({ movies: sortedMovies })
+    } 
+  }
+  
   render() {
+    console.log(this.state.movies)
     return (
       <>
-        <h1 className="cards-title">All movies</h1>
-        <section className="card-area">
-        {this.injectMovies()}
-        </section> 
-      </>
-    )
+      <SortMovies sortMovies={this.sortMovies}/>
+      <section className="card-area">
+      {this.injectMovies(this.state.movies)}
+      </section> 
+    </>
+  )
   }
 }
+
+/*
+average_rating: 9
+backdrop_path: "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg"
+id: 694919
+poster_path: "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg"
+release_date: "2020-09-29"
+title: "Money Plane"
+*/
```

Comment:
Don't forget to add in `PropTypes` as a form of type checking to your components!